core/src/validator.rs

```diff
@@ -175,6 +176,7 @@ pub struct ValidatorConfig {
     pub ledger_column_options: LedgerColumnOptions,
     pub runtime_config: RuntimeConfig,
     pub replay_slots_concurrently: bool,
+    pub banking_trace_size: u64,
```

Comment:
What does size mean in this context?
Couple of thoughts pop in: Filesize, Number of files, Total disk size?

Reply:
i thought hard to disambiguate this with proper naming: db4732c1efec119e47e23e880bf2c7b8e1742d04

as for option, i avoided unclear semantics difference between `None` and `Some(0)`. Definitely, i don't want `Some(0)` to mean _no limit_ (i.e. writes trace events indefinitely).

Reply:
Yeah, I also don't think `Some(0)` should mean no limit. IIRC, w/ the current implementation it would cause an error to be thrown as the size is too small, which seems like the correct thing to me.